test/watcher.js

```diff
@@ -566,9 +553,15 @@ group('chokidar is installed', function (beforeEach, test, group) {
 	test('initial exclude patterns override whether something is a test file', function (t) {
 		t.plan(2);
 
+		avaFiles = function (files, sources) {
+			var ret = new AvaFiles(files, sources);
+			// TODO(@jamestalmage, @novemberborn): There is no way for users to actually set exclude patterns yet.
```

Comment:
I dunno. We have `defaultExcludes`, but that gets passed to watcher without modification. You had a test for watcher that assumed it "could" be modified at some point. We just don't have any logic for allowing users to do that. I just copied the test.

If there's no way for a user to modify it, is there a point to a test that verifies behavior when it's modified?